src/chat/handler.rs

```diff
@@ -0,0 +1,38 @@
+use slack;
+
+pub struct Handler;
+
+impl slack::EventHandler for Handler {
+
+    fn on_event(&mut self, cli: &slack::RtmClient, event: slack::Event) {
+        println!("on_event");
+
+        if let slack::Event::Message(_message) = event {
+            // find the testing_blursed_bot channel id
+            // TODO Tidy this up
+            let testing_channel_id = cli.start_response()
+                .channels
+                .as_ref()
+                .and_then(|channels| {
+                              channels
+                                  .iter()
+                                  .find(|chan| match chan.name {
+                                            None => false,
+                                            Some(ref name) => name == "testing_blursed_bot",
+                                        })
+                          })
+                .and_then(|chan| chan.id.as_ref())
+                .expect("testing_blursed_bot channel not found");
+
+            let _ = cli.sender().send_message(&testing_channel_id, "Hello, I'm a blursed bot");
+        }
```

Comment:
This is adapted from the `slack-rs` example code at https://github.com/slack-rs/slack-rs/blob/master/examples/slack_example.rs. We can tidy it up in various ways, but for now it's just a first working example.